config/examples/stilts_example/mrpc_mnli_phase1.conf

```diff
@@ -0,0 +1,36 @@
+include "../../defaults.conf"
+
+// For STILTS training, first run this experiment, before 
+// mrpc_mnli_phase2.conf.In this STILTS experiment, we add first train on MRPC, 
```

Comment:
Fix spacing

Reply:
Why MRPC then QQP then MRPC? This doesn't make sense (MRPC is way too small to be useful for pretraining), and it doesn't match the filename...

Reply:
Maybe replace this with something like MNLI => RTE? That was the simplest success case for STILTs.

If you'd rather stick closer to this, maybe switch to just BERT+QQP => MRPC, which is at least fairly likely to help on the target task.

Reply:
Just to make sure, this is RTE pretraining, then intermediate task of MNLI, and then finetuning on RTE?

Reply:
Simper: Train BERT on MNLI, then on RTE.

Reply:
Probably doable with a single conf.

Reply:
Definitely should be a single run / configuration. Something like:
`pretrain_tasks = mnli`
`target_tasks = rte`
Would do it.